docs/data-masking-function-list.md

```diff
@@ -24,14 +43,38 @@ The feature is in [tech preview](glossary.md#tech-preview).
 | [`mask_ssn(str [,mask_char])`](#mask_ssnstr-mask_char)    | Masks the US Social Security number                   |
 | [`mask_uk_nin(str [,mask_char])`](#mask_uk_ninstr-mask_char)     | Masks the United Kingdom National Insurance number    |
 | [`mask_uuid(str [,mask_char])`](#mask_uuidstr-mask_char)         | Masks the Universally Unique Identifier               |
+| [`masking_dictionaries(str)`](#masking_dictionariesstr) | Set a different database name to use for the dictionaries table. |
+| [`masking_dictionaries_flush()`](#masking_dictionaries_flush) | Resyncs the internal dictionary term cache |
 | [`masking_dictionary_remove(dictionary_name)`](#masking_dictionary_removedictionary_name)          | Removes the dictionary                                |
 | [`masking_dictionary_term_add(dictionary_name, term_name)`](#masking_dictionary_term_adddictionary_name-term_name)     | Adds a term to the masking dictionary                 |
 | [`masking_dictionary_term_remove(dictionary_name, term_name)`](#masking_dictionary_term_removedictionary_name-term_name)      | Removes a term from the masking dictionary            |
 
+
+## dictionaries_flush_interval_seconds(integer, unsigned)
+
+The number of seconds between a synchronization between the dictionaries table and the internal dictionary cache.
+
+This variable is read-only. Its default value is 0, which means the synchronization operation does not run. 
+
+## Version update
+
+Percona Server for MySQL 8.0.41 adds this variable.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| `seconds` | Yes | The number of seconds between a synchronization of the dictionary internal cache and dictionaries table.  | Integer, unsigned |
+
+
 ## gen_blocklist(str, from_dictionary_name, to_dictionary_name)
 
 Replaces a term from one dictionary with a randomly selected term in another dictionary.
 
+### Version update
+  
+Percona Server for MySQL 8.0.41 introduces an internal term cache. The server now uses in-memory data structures for lookups instead of querying the `mysql.masking_dictionaries` table every time. This improvement boosts performance, especially when handling multiple rows.
```

Comment:
```suggestion
Percona Server for MySQL 8.0.41 introduces an internal term cache. The server now uses in-memory data structures for lookups instead of querying the `<masking_functions.masking_database>.masking_dictionaries` table every time. This improvement boosts performance, especially when handling multiple rows.
```